### DISCUSSION TITLE
Borders and Tiling feature

### DISCUSSION THREAD

**Discussion by danielradosa** [rxhanson/Rectangle#1204] [Ideas]
Hey, I think it would be great idea if Rectangle would contain also the option to set borders, their color and thickness. Along with tiling, like Fibonnaci for example. I really appreciate that functionality on suckless st terminal. It would be awesome to have it in Rectangle as well.
Or even Rectangle Pro. That'd be something I'd pay for.
Because right now I don't see much difference with the pro version.
And it'd also be great if Rectangle could remember if window was centered and put to fullscreen when reopened. Because currently no matter the app, if I open new window for it — it's like 15px off to right and down. Or something like that. My kitty terminal does it the other way. So it would be great if all windows by default lets say opened centered and maximized. (not fullscreen)

**Comment by rxhanson** [comment]
Thanks for the feedback. 

> Rectangle would contain also the option to set borders, their color and thickness

In macOS, window properties like this are not adjustable by other apps. Rectangle could do this with its own windows, like the prefs window or footprint window, but not windows belonging to other apps.

> Tiling

This is something I would like to add to Rectangle Pro. There are limitations in macOS that would have to be worked around, but at some point I will give it a try.

> And it'd also be great if Rectangle could remember if window was centered and put to fullscreen when reopened.

This would require polling macOS for all window information (there is no notification for this), and that's why it's not currently in Rectangle/Pro. This is another one that I'd like to add in at some point if I can come up with a good solution for it.

> Because right now I don't see much difference with the pro version.

I read this as "Rectangle Pro doesn't have the features I'm looking for". There is actually a lot in there, especially compared to any other macOS window management app I've come across. I'm always looking to improve it though, and have a large backlog of features I would like to add in.
